web/client/components/misc/datetimepicker/DateTimePicker.js

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Calendar } from 'react-widgets';
+import localizer from 'react-widgets/lib/localizers/moment';
+import { Tooltip } from 'react-bootstrap';
+import OverlayTrigger from '../OverlayTrigger';
+import Hours from './Hours';
+
+localizer(moment);
+
+// lang is supported by moment < 2.8.0 in favour of locale
+const localField = typeof moment().locale === 'function' ? 'locale' : 'lang';
+
+function getMoment(culture, value, format) {
+    return culture ? moment(value, format)[localField](culture) : moment(value, format);
+}
+
+const setTime = (date, dateWithTime) => {
+    const value = moment(date);
+    value.hours(dateWithTime.getHours())
+        .minute(dateWithTime.getMinutes())
+        .seconds(dateWithTime.getSeconds())
+        .milliseconds(dateWithTime.getMilliseconds());
+    return value.toDate();
+};
+
+const formats = {
+    base: 'lll',
+    date: 'L',
+    time: 'LT'
+};
+
+class DateTimePicker extends Component {
```

Comment:
This DateTimePicker supports also the operator. Please document this a little bit.